mobx/lib/src/api/observable_collections/observable_set.dart

```diff
@@ -22,25 +22,10 @@ class ObservableSet<T>
     implements
         Listenable<SetChange<T>> {
   ObservableSet({ReactiveContext? context, String? name})
-      : this._(context ?? mainContext, HashSet(), name);
+      : this._(context ?? mainContext, LinkedHashSet<T>.identity(), name);
```

Comment:
Why "identity" here? Can we use `Set()`?

Reply:
Yep `Set()` = `LinkedHashSet.identity()` so we can use any of these.

Reply:
Well, seems no. `Set() = LinkedHashSet()`. `Set.identity() = LinkedHashSet.identity()`.

Reply:
Yes you're right I mis understood the identical comparer. Will push a hotfix, should be `Set()` here.
Thanks for that.

Reply:
You are welcome :)